iohub/display_utils.py

```diff
@@ -7,20 +7,22 @@
 
 """ Dictionary with key works and most popular fluorescent probes """
 CHANNEL_COLORS = {
+    # emission around 510-525 nm
     "lime": ["GFP", "Green", "Alexa488", "GCaMP", "FITC", "mNeon"],
+    # emission around 580 - 610 nm
     "magenta": [
         "TXR",
         "RFP",
         "mScarlet",
         "mCherry",
         "dTomato",
-        "Cy5",
         "Alexa561",
     ],
+    # emission around 440 - 460 nmm
     "blue": ["DAPI", "Blue", "BFP"],
     "red": ["Red"],
-    "orange": ["Orange", "Cy3"],
-    "yellow": ["Alexa561"],
+    "yellow": ["Cy3"],  # Cy3 emission is at 570 nm
```

Comment:
Since we are at it, should we add the color names as values of the color keys? i.e can we add `yellow` as an option for `yellow` and for whatever color?

Reply:
`Alexa561` also needs to be yellow then.